src/chem/viirs_aod2ioda.py

```diff
@@ -10,42 +11,74 @@
 import argparse
 import netCDF4 as nc
 import numpy as np
-from datetime import datetime, timedelta
+from datetime import datetime, date, timedelta
```

Comment:
None of these are actually used... how are you computing the time for each observation?

Reply:
Good catch @CoryMartin-NOAA. After looking in the test reference file (`test/testoutput/viirs_aod.nc`) there are no time data being generated, and as a result the ioda reader will not accept this file. (Every ioda file needs to contain: MetaData/latitude, MetaData/longitude and MetaData/datetime variables.)

The modis_aod2ioda.py script is writing MetaData/datetime to the output file, so perhaps you can copy that code for this script.

Thanks for taking care of this.